3723/2/.env

```diff
@@ -0,0 +1 @@
+TOKEN=Add your token here
```

Comment:
https://github.com/bkeepers/dotenv#should-i-commit-my-env-file

Reply:
And add .env into gitignore

Reply:
@stahor Done, hope now i did it right